tests/test_merger.py

```diff
@@ -117,6 +119,7 @@ def check_outline(tmp_path):
         "Bar",
         "Baz",
         "foo",
+        "An outline item",  # this has been moved to end normal???
```

Comment:
I think this is something where I need to have a closer look. This could indicate an issue.

Reply:
isolated into #1411